test/e2e/util/util.go

```diff
@@ -89,13 +91,13 @@ func createTester() *Tester {
 	}
 	clusterID1, err := getClusterID(clientset1, namespace)
 	if err != nil {
-		klog.Errorf("an error occurred while getting cluster-id configmap %s", err)
-		os.Exit(1)
+		klog.Warningf("an error occurred while getting cluster-id configmap %s", err)
+		clusterID1 = ""
```

Comment:
why don't you return an error? can we work properly without clusterID?

Reply:
yes, after uninstall I have to access standard Kubernetes resource (as pods and nodes), but cluster-id doesn't exist anymore

Reply:
ok I get it